gcc/testsuite/rust/rustc/README.md

```diff
@@ -0,0 +1,4 @@
+This repository contains test cases from the 
+[rustc test suite](https://github.com/rust-lang/rust/tree/master/tests). The
+conversion of these tests into the DejaGnu format is done by the rustc 
+testsuite adaptor, a tool specifically designed for this purpose.
```

Comment:
Later we'll need to mention where could people find this tool.

Reply:
Sure, I'll open a new issue. So, that we won't forgot about this :)